products/jbrowse-web/src/jbrowseModel.js

```diff
@@ -52,6 +52,9 @@ export default function JBrowseWeb(
       // track configuration is an array of track config schemas. multiple
       // instances of a track can exist that use the same configuration
       tracks: types.array(pluginManager.pluggableConfigSchemaType('track')),
+      textSearchAdapters: types.array(
```

Comment:
the adapters here are the aggregate ones, right? should they have aggregate in their name? I'm genuinely not sure

Reply:
yeah these are the aggregate text search adapter. Sounds good. Renames them to aggregateTextSearchAdapters